src/string_util.h

```diff
@@ -37,8 +37,9 @@ inline std::string StrCat(Args&&... args) {
   return ss.str();
 }
 
-void ReplaceAll(std::string* str, const std::string& from,
-                const std::string& to);
+struct StrEscape : std::string {
```

Comment:
Uhm.
/me doesn't like

Reply:
ooh i didn't spot that. yeah, that should likely just be a method that takes a string and returns a string.

Reply:
/what /does /you /prefer /?

Reply:
how about this...